library/core/src/num/nonzero.rs

```diff
@@ -1543,33 +1543,25 @@ macro_rules! nonzero_integer_signedness_dependent_methods {
         #[rustc_const_unstable(feature = "isqrt", issue = "116226")]
         #[must_use = "this returns the result of the operation, \
                       without modifying the original"]
-        #[inline]
+        #[inline(always)]
```

Comment:
```suggestion
        #[inline(always)] // Inline the wrapper, the inner routine can outline
```

To be consistent with https://github.com/rust-lang/rust/pull/128166/files#r1722390704

Reply:
Change to `#[inline]`, don't worry about the comment (https://github.com/rust-lang/rust/pull/128166#discussion_r1730415097)